server/webapp/WEB-INF/rails/webpack/views/pages/elastic_profiles/modals.tsx

```diff
@@ -272,12 +272,12 @@ export class UsageElasticProfileModal extends Modal {
     }
 
     const data = this.usages.map((usage) => [
-      usage.pipelineName(),
-      usage.stageName(),
-      usage.jobName(),
+      <span className={styles.cell}>{usage.pipelineName()}</span>,
+      <span className={styles.cell}>{usage.stageName()}</span>,
+      <span className={styles.cell}>{usage.jobName()}</span>,
```

Comment:
not sure if `cell` clarifies the intention. for now, can we change it to `table-cell`?